modules/gapi/include/opencv2/gapi/infer.hpp

```diff
@@ -636,11 +636,38 @@ infer2(const std::string& tag,
     return cv::GInferListOutputs{std::move(call)};
 }
 
+// NB: Functions defined below used for python.
```

Comment:
are used for Python bindings.

Reply:
...but does it make to move it to another namespace then?

Also, does it make sense to define it here or in the python bindings header?

Reply:
> ...but does it make to move it to another namespace then?
> 
> Also, does it make sense to define it here or in the python bindings header?

Make sense to move it in the python bindings header, but not change namespace, otherwise it'll reflected on python API:
```
cv.gapi.python.infer(...) - looks ugly
cv.gapi.infer(...) - looks ok
```

Reply:
Done